corehq/apps/app_manager/models.py

```diff
@@ -2180,7 +2180,7 @@ class CaseSearchProperty(DocumentSchema):
 class DefaultCaseSearchProperty(DocumentSchema):
     """Case Properties with fixed value to search on"""
     property = StringProperty()
-    default_value = StringProperty(exclude_if_none=True)
+    defaultValue = StringProperty(exclude_if_none=True)
```

Comment:
Where is this in use already? Are there already apps that use `defaultValue`?

Reply:
It's odd I know but all apps use `defaultValue`, none use `default_value`.

The JS passes `defaultValue` to the view: https://github.com/dimagi/commcare-hq/blob/1c0e8c925acd19f6c8ae150ee6e6ffbc7efedfae/corehq/apps/app_manager/static/app_manager/js/details/case_claim.js#L421-L424 which get's set directly on the class: https://github.com/dimagi/commcare-hq/blob/07ce3a92800379c385f64955114d21532fb79e4d/corehq/apps/app_manager/views/modules.py#L1278-L1281

You can see here were it's used in `RemoteRequestFactory` that we use `defaultValue`: https://github.com/dimagi/commcare-hq/blob/77b186e2fc2e53328ed30c20a27b4ce6bb0df322/corehq/apps/app_manager/suite_xml/post_process/remote_requests.py#L200

All the test etc. already use `defaultValue`.

Reply:
Also just noting that I did test that saving default values from the UI still works as does the suite generation.

Reply:
argh...yeah, that's weird. Thank you for laying put how it happens.